eras/shelley/impl/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.17.0.0
 
+* Deprecate `Shelley.BlockChain` in favour of `Shelley.BlockBody`. #5156
```

Comment:
```suggestion
* Deprecate `Cardano.Ledger.Shelley.BlockChain` in favour of `Cardano.Ledger.Shelley.BlockBody`. #5156
```

It is funny, my browser knows that I am in the US and suggests me that the British spelling of the word "favor" is wrong. :rofl: 
<img width="1180" height="196" alt="image" src="https://github.com/user-attachments/assets/a8371a8c-1960-4581-b9ee-d4ee143d5c7b" />